<!-- Improved compatibility of back to top link: See: https://github.com/othneildrew/Best-README-Template/pull/73 -->
<a name="readme-top"></a>

<!-- PROJECT LOGO -->
<br />
<div align="center">

<h3 align="center">Learning phylogenetic trees from multiple sequence alignment of the COVID data ðŸŒ³</h3>

  <p align="center">
    Department of Bioinformatics, IBB, University of Tehran
    <br />
  </p>
</div>

<div align="center">
  <img src="LOGO.PNG" alt="Alt text">
</div>

<!-- TABLE OF CONTENTS -->
<details>
  <summary>Table of Contents</summary>
  <ol>
    <li>
      <a href="#about-the-project">Data preparation </a>
      <ul>
      </ul>
    </li>
    <li>
      <a href="#about-the-project">Phylogenetic analysis</a>
      <ul>
      </ul>
    </li>
	<li>
      <a href="#about-the-project">Data visualization </a>
      <ul>
      </ul>
    </li>
  </ol>
</details>

<!-- ABOUT THE PROJECT -->
## ROADMAP

The objective of this project is to investigate the evolutionary history of a virus strain by performing a phylogenetic analysis. To accomplish this, you will use the "MAFFT" and "RAxML-NG" tools to estimate the topology and parameters of the phylogenetic tree using the FASTA file data of Omicron, Delta, or Alpha strain genomes. 
<p align="right">(<a href="#readme-top">back to top</a>)</p>

## Languages Used ðŸ“š

 R Packages 
<p align="right">(<a href="#readme-top">back to top</a>)</p>

<!-- CONTACT -->
## Contact ðŸ“§

Fereshteh Noroozi - [@LinkedIn](https://ir.linkedin.com/in/fereshteh-noroozi-a90886118?original_referer=https%3A%2F%2Fwww.google.com%2F) 

<p align="right">(<a href="#readme-top">back to top</a>)</p>

<!-- ACKNOWLEDGMENTS -->
## Acknowledgments ðŸ™Œ

Department of Bioinformatics, IBB, University of Tehran

<p align="right">(<a href="#readme-top">back to top</a>)</p>

---

<!-- Improved compatibility of back to top link: See: https://github.com/othneildrew/Best-README-Template/pull/73 -->
<a name="readme-top"></a>

<!-- PROJECT LOGO -->
<br />
<div align="center">

<h3 align="center">Protein-Structure-Modeling-Modeller ðŸ§¬</h3>

  <p align="center">
    Department of Bioinformatics, IBB, University of Tehran
    <br />
  </p>
</div>

<div align="center">
  <img src="LOGO.PNG" alt="Alt text">
</div>

<!-- TABLE OF CONTENTS -->
<details>
  <summary>Table of Contents</summary>
  <ol>
    <li>
      <a href="#about-the-project">Before you begin, make sure you have the FASTA files of your template sequences</a>
      <ul>
    </li>
    <li>
      <a href="#about-the-project">Next, identify the best model to serve as a reference.</a>
      <ul>
    </li>
	<li>
      <a href="#about-the-project">Perform sequence alignment and create an 'alignment.ali' file.</a>
      <ul>
    </li>
  </ol>
</details>

<!-- ABOUT THE PROJECT -->
## ROADMAP

We aim to build a model for a new protein sequence with an unknown 3D structure. After identifying the best reference, we create models using MODELLER. For this, we require the 'reference.pdb,' 'alignment.ali,' and 'python-script.py.
<p align="right">(<a href="#readme-top">back to top</a>)</p>

## Languages Used ðŸ’»

Python, modeller shell 
<p align="right">(<a href="#readme-top">back to top</a>)</p>

<!-- GETTING STARTED -->
## Getting Started
### Command
mod10.4 modeller-script.py

<p align="right">(<a href="#readme-top">back to top</a>)</p>

<p align="right">(<a href="#readme-top">back to top</a>)</p>

<!-- CONTACT -->
## Contact ðŸ“§

Fereshteh Noroozi - [@LinkedIn](https://ir.linkedin.com/in/fereshteh-noroozi-a90886118?original_referer=https%3A%2F%2Fwww.google.com%2F) 

<p align="right">(<a href="#readme-top">back to top</a>)</p>

<!-- ACKNOWLEDGMENTS -->
## Acknowledgments ðŸ™Œ

Department of Bioinformatics, IBB, University of Tehran

<p align="right">(<a href="#readme-top">back to top</a>)</p>

---

<!-- Improved compatibility of back to top link: See: https://github.com/othneildrew/Best-README-Template/pull/73 -->
<a name="readme-top"></a>

<!-- PROJECT LOGO -->
<br />
<div align="center">

<h3 align="center"> Bayesian-Network-Analysis ðŸ“Š</h3>

  <p align="center">
    Department of Bioinformatics, IBB, University of Tehran
    <br />
  </p>
</div>

<div align="center">
  <img src="Rplot-beysian-network.png" alt="Alt text">
</div>

## Languages Used ðŸ“š

R
<p align="right">(<a href="#readme-top">back to top</a>)</p>

### Installation

packages: igraph and bnlearn.

<p align="right">(<a href="#readme-top">back to top</a>)</p>

<p align="right">(<a href="#readme-top">back to top</a>)</p>

<!-- CONTACT -->
## Contact ðŸ“§

Fereshteh Noroozi - [@LinkedIn](https://ir.linkedin.com/in/fereshteh-noroozi-a90886118?original_referer=https%3A%2F%2Fwww.google.com%2F) 

<p align="right">(<a href="#readme-top">back to top</a>)</p>

<!-- ACKNOWLEDGMENTS -->
## Acknowledgments ðŸ™Œ

Department of Bioinformatics, IBB, University of Tehran

<p align="right">(<a href="#readme-top">back to top</a>)</p>

---

<!-- Improved compatibility of back to top link: See: https://github.com/othneildrew/Best-README-Template/pull/73 -->
<a name="readme-top"></a>

<!-- PROJECT LOGO -->
<br />
<div align="center">

<h3 align="center">Read mapping and genome assembly
